features/src/main/scala/com/salesforce/op/utils/spark/OpVectorColumnMetadata.scala

```diff
@@ -151,6 +151,12 @@ case class OpVectorColumnMetadata // TODO make separate case classes extending t
     if (hasParentOfSubType[OPMap[_]]) parentFeatureName.map(p => grouping.map(p + "_" + _).getOrElse(p))
     else parentFeatureName
 
+  /**
+   * Get the feature grouping qualified by the parent feature name
+   * @return Optional string of feature grouping
+   */
+  def featureGroup(): Option[String] = grouping.map(g => s"${parentFeatureName.mkString("_")}_$g")
```

Comment:
@leahmcguire repeating my previous comment here -  dont you want to do something similar to `s"${parentFeatureName.mkString("_")}${grouping.map("_" + _).getOrElse("")}"` (same as we do in [`makeColName()`](https://github.com/salesforce/TransmogrifAI/blob/master/features/src/main/scala/com/salesforce/op/utils/spark/OpVectorColumnMetadata.scala#L125) function)

Reply:
Especially because you do call `.get` here - https://github.com/salesforce/TransmogrifAI/pull/153/files#diff-55c6247553ff142cb1657c1bac3c728fR459

Reply:
no because before calling that get there is a check on whether this is a None which determines if the categorical stats are computed

Reply:
actually the check is directly on the group - I will change this